### ISSUE TITLE
Update IPT stats

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#241]
**Update IPT stats**

Latest stats are available from http://www.gbif.org/ipt 

Perhaps a simple tagline such as &quot;Used by more than 600 publishers worldwide to publish 3600 datasets and counting.. &quot; would do fine instead of the breakdown by pubilshers, installations and countries. Ultimately, we want to promote the reuse of existing installations, not try and incentivize new installations popping up around the globe.

-----
fbitem-7139f8609c88dfc8d6f63e185b48f05341718691
Reported by: @kbraak
System: Firefox 54.0.0 / Mac OS X 10.10.0
Referer: https://demo.gbif.org/ipt
Window size: width 1255 - height 847
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-07-10T14:14:37.895Z',mode:absolute,to:'2017-07-10T14:20:37.895Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-07-10T14:14:37.895Z',mode:absolute,to:'2017-07-10T14:20:37.895Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by MortenHofft** [comment]
@kbraak is there an API for this or do you manually change numbers from time to time?

**Comment by kbraak** [comment]
They are generated on demand using [this code](https://github.com/gbif/ipt/blob/master/src/test/java/org/gbif/ipt/service/registry/impl/RegistryWsClientTest.java#L64). Thanks.

**Comment by MortenHofft** [comment]
Is it something you believe would make sense to expose as an endpoint then?

**Comment by kbraak** [comment]
Takes a long time to run is the problem, otherwise yea that would be nice.

**Comment by MortenHofft** [comment]
how long is long? Couldn't it serve the latest fast and then do a weekly update for example. how much would it require not having to do it manually you think?

**Comment by kbraak** [comment]
Roughly ten minutes last time I checked.

If such an IPT stats endpoint was added to the GBIF API, it would certainly require a significant number of changes across several projects (API, Registry), plus an API documentation update. Maybe @fmendezh can suggest alternate implementations though?

I have collected multiple years of IPT stats in Excel and use these to generate charts, for example showing the growth of occurrence datasets published by IPTs over time. If we wanted to produce these types of IPT charts, I suppose we'd need to also start storing these stats in the Registry database for example.
